### PR TITLE
Homework 5 classes and objects

### PR DESCRIPTION
**Pull request by AntTurieva**
Homework 5 classes and objects

### CODE REVIEW

**Comment by AntBelyi** [comment]
Where are all you merged tasks?
Screenshot_1

File: src/main/java/hw_5/myCity/Building.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package hw_5.myCity;
+
+
+import hw_5.myClass.Person;
+
+public class Building {
+    public Person[] persons;
+
+
```
Comment (AntBelyi) [inline src/main/java/hw_5/myCity/Building.java:9]: Please add some information to Person class(name, surname, age, toString()), create few persons in main method in Building class, add these persons to array and using cycle display persons from array

File: src/main/java/hw_5/Task2/MyInt.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package hw_5.Task2;
+
+public class MyInt {
+    public int x;
+
+
```
Comment (AntBelyi) [inline src/main/java/hw_5/Task2/MyInt.java:6]: Please remove extra line

File: src/main/java/hw_5/Task2/MyInt.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    public MyInt(int x) {
+        this.x = x;
+    }
+    @Override
```
Comment (AntBelyi) [inline src/main/java/hw_5/Task2/MyInt.java:10]: there is should be extra empty line between 9 and 10 rows

File: src/main/java/MainProgram.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+        System.out.println(a1.x);
+        System.out.println(a2.x);
+
+        a1.x = 5;
+
+        System.out.println(a1.x);
+        System.out.println(a2.x);
+
+
```
Comment (AntBelyi) [inline src/main/java/MainProgram.java:16]: I guess this "MainProgram" class should be located in hw_5 folder?

**Comment by AntBelyi** [comment]
I don't see all your merged tasks
image

File: src/main/java/hw_5/myCity/Building.java (hunk 1 above)
Comment (AntTurieva) [reply src/main/java/hw_5/myCity/Building.java:9]: fixed

File: src/main/java/hw_5/Task2/MyInt.java (hunk 2 above)
Comment (AntTurieva) [reply src/main/java/hw_5/Task2/MyInt.java:6]: fixed

File: src/main/java/hw_5/Task2/MyInt.java (hunk 3 above)
Comment (AntTurieva) [reply src/main/java/hw_5/Task2/MyInt.java:10]: fixed

File: src/main/java/MainProgram.java (hunk 4 above)
Comment (AntTurieva) [reply src/main/java/MainProgram.java:16]: fixed

**Comment by AntTurieva** [comment]
fixed

**Pull request closed without merging**